docs/source/core_services/security/Key-Stores.rst

```diff
@@ -0,0 +1,45 @@
+.. _Key-Stores:
+==========
+Key Stores
+==========
+
+*Note: most VOLTTRON users should not need to directly interact with
+agent key stores. These are notes for VOLTTRON platform developers.
+This is not a stable interface and the implementation details are 
+subject to change.*
+
+Each agent has its own encryption key-pair that is used to
+:ref:`authenticate<VIP-Authentication>` itself to the VOLTTRON
+platform. A key-pair comprises a public key and a private (secret) key.
+These keys are saved in a key store, which is implemented by the
+:doc:`KeyStore class<volttron.platform.keystore.KeyStore>`.
+Each agent has its own key store.
+
+Key Store Locations
+-------------------
+
+There are two main locations key stores will be saved. Installed agents'
+key stores are in the the agent's data directory::
+
```

Comment:
Don't placeholders in man pages use less than greater than signs around the placeholder?